stetho/src/test/java/com/facebook/stetho/inspector/elements/android/MethodInvokerTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.elements.android;
+
+import android.app.Activity;
+import android.os.Build;
+import android.widget.TextView;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.assertEquals;
+
+@Config(emulateSdk = Build.VERSION_CODES.JELLY_BEAN)
+@RunWith(RobolectricTestRunner.class)
+public class MethodInvokerTest {
+
+  private final TextView mTextView = new TextView(Robolectric.setupActivity(Activity.class));
```

Comment:
let's also test something other than `TextView` ... like create a `CheckBox` and invoke `setChecked`